pkg/kn/commands/root.go

```diff
@@ -84,6 +112,18 @@ func initConfig() {
 
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err == nil {
```

Comment:
unrelated to the PR, but we should exit if ReadInConfig fails. ideally though all the exits are consolidated via return err.

Reply:
ok